x-pack/plugins/security_solution/server/lib/detection_engine/rules/get_prepackaged_rules.ts

```diff
@@ -16,10 +16,27 @@ import {
   AddPrepackagedRulesSchemaDecoded,
 } from '../../../../common/detection_engine/schemas/request/add_prepackaged_rules_schema';
 import { BadRequestError } from '../errors/bad_request_error';
+import {
+  getAsset,
+  getPathParts,
+  getRegistryPackage,
+  fetchFindLatestPackage,
+} from '../../../../../fleet/server';
 
-// TODO: convert rules files to TS and add explicit type definitions
 import { rawRules } from './prepackaged_rules';
 
+const DetectionRulesPackageName = 'detection_rules';
+let latestRulesPackageVersion: string | undefined;
+let latestRulesDownload: AddPrepackagedRulesSchemaDecoded[];
+let usePackageRegistryRules = false;
+
+/**
+ * Enable retrieval of rules from the Elastic Package Registry.
+ */
+export const enablePackageRegistryRules = () => {
+  usePackageRegistryRules = true;
```

Comment:
I would do a `return process.env.ENABLE_REGISTRY_RULES || false` here and then we will have it by default turned off. In your local .bashrc/.zshrc we can flip it to true.

Reply:
Oh wait, never mind, I see what you're doing here. You're using the config globally and then flipping it here from the plugin. You can do this for now, I'm fine with it. Usually we push the config object with the feature flags in the REST routes. I will see if I can give a pointer there for you, but this is fine right now as written. It will work out.

Reply:
so entirely remove it from the config then?
followed by a change from L31-L37, removing this function and instead:
```typescript
const usePackageRegistryRules = process.env.ENABLE_REGISTRY_RULES || false;
```